pkg/blockatlas/marshal_test.go

```diff
@@ -75,3 +75,25 @@ func TestTx_MarshalJSON(t *testing.T) {
 	// Compare expected and output JSON
 	bytes.Equal(got, txJSON)
 }
+
+func TestTxPage_Sort(t *testing.T) {
+	tests := []struct {
+		name string
+		txs  TxPage
+		want TxPage
+	}{
+		{
+			"sort 1",
+			TxPage{{Date: 9999}, {Date: 46}, {Date: 100}, {Date: 1}, {Date: 777}},
+			TxPage{{Date: 1}, {Date: 46}, {Date: 100}, {Date: 777}, {Date: 9999}},
```

Comment:
I think we need transactions to be sorted in the opposite way? 

Since it's a unix timestramp we need them to be sorted from old to new?